placenames.mss

```diff
@@ -31,56 +31,127 @@
 }
 
 .state {
-  [admin_level = '4'] {
-    [zoom >= 4][zoom < 5][way_pixels > 750],
-    [zoom >= 5][way_pixels > 3000][way_pixels < 196000] {
-      text-name: "[ref]";
-      text-size: 9;
-      text-fill: @state-labels;
-      text-face-name: @oblique-fonts;
-      text-halo-fill: @standard-halo-fill;
-      text-halo-radius: @standard-halo-radius * 1.5;
-      text-wrap-width: 0;
-      text-placement: interior;
-      [zoom >= 5] {
-        text-name: "[name]";
-        text-wrap-width: 30;
-      }
-      [zoom >= 7] {
-        text-size: 11;
-        text-wrap-width: 50;
-      }
+  [zoom >= 4][zoom < 5][way_pixels > 750],
+  [zoom >= 5][way_pixels > 3000][way_pixels < 196000] {
+    text-name: "[ref]";
+    text-size: 9;
+    text-fill: @state-labels;
+    text-face-name: @oblique-fonts;
+    text-halo-fill: @standard-halo-fill;
+    text-halo-radius: @standard-halo-radius * 1.5;
+    text-wrap-width: 0;
+    text-placement: interior;
+    [zoom >= 5] {
+      text-name: "[name]";
+      text-wrap-width: 30;
+    }
+    [zoom >= 7] {
+      text-size: 11;
+      text-wrap-width: 50;
+    }
+  }
+}
+
+#capital-names {
+  [zoom >= 4][zoom < 8][population > 600000],
+  [zoom >= 5][zoom < 8] {
+    shield-file: url('symbols/place/place-capital-6.svg');
+    shield-text-dx: 6;
+    shield-text-dy: 6;
+    shield-name: '[name]';
+    shield-face-name: @book-fonts;
+    shield-fill: @placenames;
+    shield-size: 11;
+    shield-margin: 5;
+    shield-avoid-edges: true;
+    shield-wrap-width: 30;
+    shield-halo-fill: @standard-halo-fill;
+    shield-halo-radius: @standard-halo-radius * 1.5;
+    shield-placement-type: simple;
+    shield-placements: 'S,N,E,W';
+    [dir = 1] {
+      shield-placements: 'N,S,E,W';
+    }
+    shield-unlock-image: true;
+
+    [zoom >= 5] {
+      shield-wrap-width: 45;
+    }
+    [zoom >= 6] {
+      shield-size: 12;
+      shield-wrap-width: 60;
+    }
+    [zoom >= 7] {
+      shield-file: url('symbols/place/place-capital-8.svg');
+      shield-text-dx: 7;
+      shield-text-dy: 7;
+    }
+  }
+  [zoom >= 8] {
```

Comment:
Does this mean that all capitals are rendered up to z15 inclusive (as max-zoom in yaml file declares)?

If so, might it be possible to let vanish big capitals with high population count (Rome, Berlin) earlier, while small capitals (Monaca, Andorra le Vella) valish later?